stable/concourse/values.yaml

```
@@ -1101,13 +1101,15 @@ concourse:
     volumeSweeperMaxInFlight: 5
 
     tsa:
-      ## TSA host to forward the worker through.
-      ##
-      host:
 
-      ## TSA port to forward the worker through.
+      ## TSA host(s) to forward the worker through.
+      ## Only used for web-only deployments.
```

Oh, I think you meant `worker-only` deployments instead 🤔  

```suggestion
       ## Only used for worker-only deployments.
```